run-basic-batch-operators-test.sh

```diff
@@ -49,6 +56,20 @@ OPERATOR_LIST_FILENAME=operator-list.txt
 # Operator list path in the report
 OPERATOR_LIST_PATH="$REPORT_FOLDER"/"$OPERATOR_LIST_FILENAME"
 
+# Log file
+LOG_FILENAME="output_$TIMESTAMP.log"
+
+# Operator list path in the report
+LOG_FILE_PATH="$REPORT_FOLDER"/"$LOG_FILENAME"
+
+echoColor() {
+	local color=$1
+	local message=$2
+
+	echo -e "${color}$message${ENDCOLOR}"
```

Comment:
Sometimes you use `${foobar}`, and sometimes you use `$foobar`. What's the point? I think it is the same. I use `$foobar`.

Reply:
fixed